auth/user.go

```diff
@@ -183,6 +183,98 @@ func (user *userImpl) RoleHistory() TimedSetHistory {
 	return user.RoleHistory_
 }
 
+func (user *userImpl) RevokedChannels(since uint64) map[string]uint64 {
```

Comment:
Add a comment describing the contents of the returned map[string]uint64.  Even after reviewing the entire function I'm not positive what the sequence value represents (it seems like 'end seq for latest revocation', but I'm not sure how that information should be used).

Reply:
Basically the value that should be used for triggeredBy, but I'll add a comment yeah.